adal/src/main/java/com/microsoft/aad/adal/AuthenticationResult.java

```diff
@@ -89,6 +89,12 @@
 
     private Date mExtendedExpiresOn;
 
+    private String mServerErrorCode;
```

Comment:
can you box serverErrorCode, suberrorCode, refreshTokenAge and speRing into an object?

Reply:
Resolved in dff04ba44c165c0faef035488f621c90c28f7844